spotlights.html.slim

```diff
@@ -8,3 +8,6 @@
   .medium-8.columns
     a(href="http://www.redhat.com/en/about/press-releases/red-hat-launches-new-release-openshift-enterprise-and-adds-private-ipaas")
       img(src=cdn("#{site.base_url}/images/branding/hero_desktop_380x280_xpaas.png") alt="")
+  .medium-8.columns
+    a(href="http://www.jboss.org/products/bpmsuite/overview/")
+      img(src=cdn("#{site.base_url}/images/branding/hero_desktop_380x280_bpm61_03182015.jpg") alt="")
```

Comment:
You need to have exactly three items in this file as they all appear all the time, rather than with a carousel.